integration_tests/distro_configs/debian_config

```diff
@@ -0,0 +1,5990 @@
+#
```

Comment:
> How did you generate this config file?

@rodrigosiqueira 
As I mentioned, this is the .config file you sent me a while a go. 
Would we still need to maintain the config file if I implemented the use of olddefconfig? Copying the appropriate config file and optimizing it would  make the tests more complicated and blur the distro independency of the current template, so I would rather avoid it if possible